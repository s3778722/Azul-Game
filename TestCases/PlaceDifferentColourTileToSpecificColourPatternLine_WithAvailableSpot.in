1
Player1
Player2
turn 3 U 3
turn 2 R 2
turn 1 L 3