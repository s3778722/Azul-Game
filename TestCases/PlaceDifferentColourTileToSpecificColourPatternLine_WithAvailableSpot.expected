Welcome to Azul! -------------------
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> 
Starting a New Game
Enter the name for player 1:
> 
Enter the name for player 2:
> Let's Play!

=== Start Round == 
TURN FOR PLAYER:Player1
SCORE: 0
Factories:
0: F
1: Y R Y Y
2: Y R Y R
3: L L U U
4: B L R L
5: L L B B

Mosaic for Player1:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     . . . || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
6: Broken:
> 
TURN FOR PLAYER:Player2
SCORE: 0
Factories:
0: F L L
1: Y R Y Y
2: Y R Y R
3:
4: B L R L
5: L L B B

Mosaic for Player2:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     . . . || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
6: Broken:
> 

TURN FOR PLAYER:Player1
SCORE: 0
Factories:
0: F L L Y Y
1: Y R Y Y
2:
3:
4: B L R L
5: L L B B

Mosaic for Player1:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     . U U || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
6: Broken:
> Tile Couldn't be placed.


The game is a draw...... Better luck next time!!!
=== GAME OVER ===
